_posts/2023-11-09-routines.md

```diff
@@ -0,0 +1,85 @@
+documentation]({{site_url}}/docs/current/routines/examples.html), and send you
+some goodies as a reward!
+
+<!--more-->
+
+With the recent [Trino 431
+release]({{site_url}}/docs/current/release/release-431.html) we shipped a
+feature that has been awaited by many Trino users for a long, long time. [SQL
+routines]({{site_url}}/docs/current/routines.html) are an easy way to define our
+own procedural, custom functions. All users on your Trino instance can then use
+that function in their queries and enjoy the new feature to simplify their
+queries.
+
+The new process of writing a routine in your client tool in SQL can be used as
+alternative to the old way of having to create a custom plugin in Java,
+compiling it, and getting the binary deployed in your cluster. The time it takes
+to use a function has gone from hours to a few commands!
+
+Our documentation includes details for all the supported statements:
+
+* `BEGIN`
+* `CASE`
+* `DECLARE`
+* `FUNCTION`
+* `IF`
+* `ITERATE`
+* `LEAVE`
+* `LOOP`
+* `REPEAT`
+* `RETURN`
+* `SET`
+* `WHILE`
+
+With the memory connector and the Hive connector supporting routine storage, you
+can use `CREATE FUNCTION` and `DROP FUNCTION`, so that everyone using the
+cluster has access to your routines.
+
+The unit tests and our [examples
+documentation](({{site_url}}/docs/current/routines/examples.html)) contain a
+number of routines that scratch the surface of what is possible. Now, we are
+looking for you to help us improve the documentation and maybe even find some
+bugs. So here is what we are asking from you:
+
+* Upgrade your Trino cluster, CLI, and other clients to 431 or newer.
+* Learn from the documentation and write your own routines.
+* Send us your best SQL routine.
+  * Create a pull request to add to the [examples in the
+    documentation](https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/routines/examples.md)
+    with a new section, and request a review from [Manfred
+    (mosabua)](https://github.com/mosabua)
+  * Alternatively, [email the details](mailto:manfred@starburst.io) and submit a
+    [CLA](https://github.com/trinodb/cla) separately.
+* Explain the use case, what the routine does, and maybe also how it works.
+* Include the full statement for the `CREATE FUNCTION` definition and an example
+  invocation.
+* Add any necessary tables or data so we can test the function.
+* Reach out to us on the [Trino community Slack]({{site.baseurl}}/slack.html),
+  if you need any help.
+
+We plan to present submissions at [Trino Summit 2023]({% post_url
+2023-09-14-trino-summit-2023-announcement %}), write a blog post, add them to
+the documentation, and [Starburst](https://www.starburst.io/) will send a cool
+reward for the ten best entries.
+
+Also, if you have more great Trino usage to talk about and share, we would love
+to see your [speaker proposal for Trino
+Summit](https://sessionize.com/trino-summit-2023/).
+
+We look forwards to see many great tricks from you all.
```

Comment:
"look forward to seeing"

Also, why "tricks"? Maybe "submissions", "creations" ?